cli.js

```diff
@@ -3,13 +3,25 @@
 var meow = require('meow');
 var supportsColor = require('supports-color');
 
-meow({
+var cli = meow({
 	help: [
+		'Usage',
+		'  $ supports-color [--<256|16m>]',
+		'',
+		'Options',
+		'  --256     Check for 256 color support',
+		'  --16m     Check for 16 million (truecolor) support',
```

Comment:
Just curious, do you know if there's a standard for how many spaces the flag and description should have? I'm never sure.